cypress/integration/specialFeatures/cookieBanner.amp.js

```diff
@@ -33,6 +35,7 @@ Object.keys(config)
   .filter(serviceFilter)
   .forEach(service => {
     Object.keys(config[service].pageTypes)
+      .filter(shouldSmokeTest(_, service))
```

Comment:
is there buy-in for this style?  not against it, but no doubt people will have strong opinions on if we should be doing it

it's not used elsewhere in simorgh and will take people longer to understand

Reply:
+1 I think as we're trying to make this cypress codebase readable for testers we should avoid such ramda uses

Reply:
you mean the `_`? Happy to remove it if you think it will confuse people.

Reply:
+1 this is far too complicated, elegance at the cost of complexity, I vote removing rambda from cypress tests

Reply:
I spent a little while reading up on curried functions, as Drew said it's more elegant but for the majority it may be overly complex, especially with Cypress being frequently worked on by the testers.

Reply:
Apologies if the use of a placeholder confused anyone but currying is a separate pattern and I think it's worth knowing about and using in the main application code.